control/iosys.py

```diff
@@ -2030,18 +2025,28 @@ def find_eqpt(sys, x0, u0=None, y0=None, t=0, params=None,
         # Special cases: either inputs or outputs are constrained
         if y0 is None:
             # Take u0 as fixed and minimize over x
-            # TODO: update to allow discrete time systems
-            def ode_rhs(z): return sys._rhs(t, z, u0)
-            result = root(ode_rhs, x0)
+            if sys.isdtime(strict=True):
+                def state_rhs(z): return sys._rhs(t, z, u0) - z
+            else:
+                def state_rhs(z): return sys._rhs(t, z, u0)
```

Comment:
Could the difference here be documented with a comment and maybe a reference to a textbook explanation?